nuitka/utils/CStrings.py

```diff
@@ -28,34 +28,31 @@ def _encodePythonStringToC(value):
     """
     assert type(value) is bytes, type(value)
 
-    result = ""
+    special_chars = {92: r"\134", 9: r"\011", 13: r"\015", 10: r"\012", 63: r"\077", 34: r"\042"}
```

Comment:
For Python2, this would have to be different, since there `bytes` is not a number.